src/app/online-purchase/online-purchase.component.html

```diff
@@ -0,0 +1,47 @@
+<h3 class="success-message">{{msg}}</h3>
+<div class="container">
+    <h4>Online Purchase</h4>
+    <!-- Trigger the modal with a button -->
+    <label class="switch">
+        <input type="checkbox"  data-toggle="modal" data-target="#myModal" (change)="onclick()">
+        <span class="slider round"></span>
+    </label>
+    <!-- Modal -->
+    <div class="modal fade" id="myModal" role="dialog">
+        <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+                <div>
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">×</button>
+                        <h3 class="warning-msg">
+                          <span *ngIf="labelFlag">{{labelforActivateMsgHeader}}</span>
+                          <span *ngIf="!labelFlag">{{labelfoDeactivateMsgHeader}}</span>
+                        </h3>
+                    </div>
+                    <div class="modal-body">
+                        <h4 class="warning-msg">
+                            <span *ngIf="labelFlag">{{labelforActiveMsgBody}}</span>
+                            <span *ngIf="!labelFlag">{{labelforDeactivateMsgBody}}</span>
+                        </h4>
+                    </div>
+                    <div class="modal-footer">
+                        <div class="save-btn">
+                          <button  type="button"  (click)="onSaveBtn()"  style="padding: 14px 28px; background-color: #4CAF50;  width: 50%;" class="btn btn-primary" data-dismiss="modal">
+                              Save
+                          </button>
+                        </div>
+                        <br>
+                        <div class="cancel-btn">
+                            <button type="button" style="padding: 14px 28px; width: 50%;" class="btn btn-default" data-dismiss="modal">
+                                Cancel
```

Comment:
Try to add button , Title names from the constants

Reply:
add from constant file